src/Microsoft.AspNet.Server.Kestrel/Http/Frame.cs

```diff
@@ -39,6 +40,12 @@ public abstract partial class Frame : FrameContext, IFrameControl
         private static readonly byte[] _bytesDate = Encoding.ASCII.GetBytes("Date: ");
         private static readonly byte[] _bytesEndHeaders = Encoding.ASCII.GetBytes("\r\n\r\n");
 
+        private static readonly Vector<byte> _vectorCRs = new Vector<byte>((byte)'\r');
```

Comment:
It's impressive what a difference avoiding struct initialization is making at this point.